CHANGELOG/v1.18.0.md

```diff
@@ -0,0 +1,163 @@
+- Feat APIServerILB: private IP of the internal LB can be customized ([#5332](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5332), [@nawazkh](https://github.com/nawazkh))
+- Private IP of the Internal LB of the API Server will be configurable ([#5274](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5274), [@nawazkh](https://github.com/nawazkh))
+- Use community gallery for default VM images ([#5167](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5167), [@mboersma](https://github.com/mboersma))
+- Use internal LB's IP for intra-node communication ([#5209](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5209), [@nawazkh](https://github.com/nawazkh))
+
+### Documentation
+
+- Added docs describing how to migrate existing AKS clusters managed by CAPZ to the new ASO-based API ([#5305](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5305), [@nojnhuh](https://github.com/nojnhuh))
+- Overhaul Quickstart UX using AzureASOManagedCluster ([#5319](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5319), [@willie-yao](https://github.com/willie-yao))
+
+### Failing Test
+
+- Bump CAPI to v1.8.8 ([#5371](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5371), [@willie-yao](https://github.com/willie-yao))
+- Custom build templates to use curl when az login fails ([#5109](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5109), [@nawazkh](https://github.com/nawazkh))
+- Default to Marketplace images for old Kubernetes versions ([#5266](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5266), [@mboersma](https://github.com/mboersma))
+
+### Bug or Regression
+
+- Refactor(manager): set machinepool true as default ([#5146](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5146), [@KOREAN139](https://github.com/KOREAN139))
+- AKS extensions removed from an AzureManagedControlPlane's `spec.extensions` will now be deleted. ([#5139](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5139), [@nojnhuh](https://github.com/nojnhuh))
+- Fixed a bug preventing Cluster API from authenticating to ASOAPI Managed Clusters with local accounts disabled. ([#5211](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5211), [@nojnhuh](https://github.com/nojnhuh))
+- Fixed an issue where CAPZ was accumulating memory over time that could never be garbage collected. ([#5284](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5284), [@nojnhuh](https://github.com/nojnhuh))
+- Improves MachinePool scale up/down to avoid unnecessary VMSS VM model udpates ([#5164](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5164), [@mweibel](https://github.com/mweibel))
```

Comment:
```suggestion
- Improves MachinePool scale up/down to avoid unnecessary VMSS VM model updates ([#5164](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5164), [@mweibel](https://github.com/mweibel))
```

Reply:
This tripped up `codespell`, lol